src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -274,16 +284,28 @@ class HuiMapCard extends LitElement implements LovelaceCard {
         }
         // filter location data from states and remove all invalid locations
         const points = entityStates.reduce(
-          (accumulator: LatLngTuple[], entityState) => {
+          (accumulator: HaMapPathPoint[], entityState) => {
             const latitude = entityState.attributes.latitude;
             const longitude = entityState.attributes.longitude;
             if (latitude && longitude) {
-              accumulator.push([latitude, longitude] as LatLngTuple);
+              const p = {} as HaMapPathPoint;
+              p.point = [latitude, longitude] as LatLngTuple;
+              const t = new Date(entityState.last_updated);
+              if (config.hours_to_show && config.hours_to_show > 144) {
```

Comment:
we have checked this above already
```suggestion
              if (config.hours_to_show > 144) {
```

Reply:
This suggestion I initially tried, but the linter did not like it:

```
src/panels/lovelace/cards/hui-map-card.ts:294:19 - error TS18048: 'config.hours_to_show' is possibly 'undefined'.

294               if (config.hours_to_show > 144) {
```

Maybe a linter bug, but it seemed harmless to add the extra check to satisfy it.

Reply:
just tell it it can't be undefined :-)

```ts
if (config.hours_to_show! > 144)
```

Reply:
Thanks, still learning some of the syntax. 

I made this change as suggested, but it reset the approval.